### ISSUE TITLE
在 Arc 浏览器里是可以安装的，但是没法使用。

### ISSUE THREAD

**Issue by MatthewYe** [huhu/rust-search-extension#230]
是不是和 Arc 浏览器的自定义地址栏不兼容？

**Comment by shuizhongyueming** [comment]
I'm using Arc with:
Version 1.32.0 (46908)
Chromium Engine Version 122.0.6261.94

And it is still not working. Type `rs` then space or tab, cannot trigger the extension

<img width="775" alt="image" src="https://github.com/huhu/rust-search-extension/assets/2059860/a4d5bab4-f285-4595-babc-50ae4b7e66d3">

**Comment by SteveLauC** [comment]
Does anyone know what that `rs` is in Chrome's terminology? Then we can figure out why it is not supported in Arc and possibly get this fixed

**Comment by xxchan** [comment]
It is omnibox. I complained this to Arc but haven't got any response :(

**Comment by kianenigma** [comment]
Have switched to Arc today and this is so far the only feature I miss :(

**Comment by xxchan** [comment]
Please submit feedback to Arc to make it more possible for them to fix this issue..

**Comment by Folyd** [comment]
I built another brand-new product called https://query.rs, welcome to give it a try. :)

> Query.rs was inspired by Rust Search Extension. Due to the partial support of the [omnibox API](https://developer.chrome.com/docs/extensions/reference/api/omnibox) by some browsers, Rust Search Extension is not available on browsers such as [Safari](https://github.com/huhu/rust-search-extension/issues/87), [Vivaldi](https://github.com/huhu/rust-search-extension/issues/10), [Arc](https://github.com/huhu/rust-search-extension/issues/230), etc. Query.rs addresses this gap.

**Comment by xxchan** [comment]
query.rs looks quite neat! But I think one thing I loved a lot about rust-serach-extension is the ability to do search in the search bar (indeed the omnibox API), without the need to open a webside. 

So currently I switched to raycast + [dash](https://kapeli.com/dash) (with [cargo-docset](https://github.com/Robzz/cargo-docset)

**Comment by SteveLauC** [comment]
The thing I love about Rust search extension is that I can do the search locally without pulling in any network connection (I use the local mode), so query.rs won't help too much for me. 

It would be amazing if we can have Rust search extension integrated with Raycast, can image this combination will be quite attractive for macOS users🤩

**Comment by Folyd** [comment]
Raycast extension is a great idea. I'll investigate how much effort it would take to migrate Rust Search Extension to Raycast. If it's not too much, I'll do it.

BTW, query.rs will be more powerful in the future. For example, I plan to support full-text search, AI search, etc.

**Comment by SteveLauC** [comment]
>  I plan to support full-text search, AI search, etc.

You mean searching the document body, not just the Rust symbols? This is gonna be huge!

**Comment by Folyd** [comment]
> > I plan to support full-text search, AI search, etc.
> 
> You mean searching the document body, not just the Rust symbols? This is gonna be huge!

Yes, I plan to build a full-text search index for all doc texts. You'll be able to search for functions, traits, etc., if their doc text matches your keyword.
